cats/src/test/scala/magnolify/cats/ScopeTest.scala

```diff
@@ -51,3 +52,47 @@ object ScopeTest {
     ShowDerivation[Numbers]
   }
 }
+
+class ScopeTest extends FunSuite {
+
+  test("auto implicit will give most powerful abstraction") {
```

Comment:
This is how scala solves implicit conflicts.
In our case we could have only defined those as the other implicits methods can't be called explicitly due to the shapeless `LowPriority`